scripts/build/Platform/Linux/build_config.json

```diff
@@ -214,5 +214,48 @@
       "CMAKE_LY_PROJECTS": "AutomatedTesting",
       "CMAKE_TARGET": "all"
     }
+  },
+  "install_profile": {
+    "TAGS": [],
+    "COMMAND": "build_linux.sh",
+    "PARAMETERS": {
+        "CONFIGURATION": "profile",
+        "OUTPUT_DIRECTORY": "build/linux",
+        "CMAKE_OPTIONS": "-G 'Ninja Multi-Config' -DCMAKE_C_COMPILER=clang-6.0 -DCMAKE_CXX_COMPILER=clang++-6.0 -DLY_PARALLEL_LINK_JOBS=4 -DLY_DISABLE_TEST_MODULES=TRUE",
```

Comment:
This should be `clang-12` and `clang++-12` now

Reply:
will merge and update

Reply:
actually, the change went into stabilization and this one is going to development... I will talk to @burelc-amzn tomorrow to see if he wants to cherry-pick the change to development or I do it

Reply:
Sounds good.